src/shared/components/Badge/Badge.css.ts

```diff
@@ -0,0 +1,19 @@
+import { style } from '@vanilla-extract/css';
+
+export const BadgeWrapper = style({
+  padding: '0.4rem 0.5rem',
+  textAlign: 'center',
+  borderRadius: '0.4rem',
+  fontSize: '1rem',
+  fontWeight: 700,
+  lineHeight: '1.1rem',
+});
+export const red = style({
+  color: '#FF322E',
+  backgroundColor: '#FFEAEA',
+});
+
+export const blue = style({
+  color: '#4154FF',
+  backgroundColor: '#F0F1FF',
+});
```

Comment:
우리는 vanilla-extract을 사용하고 있으니까 recipes 라는 녀석을 사용해보시는 걸 추천드립니다! 
지금은 케이스가 2개지만 조건부 스타일 분기가 많아지는 컴포넌트를 작성할 때 깔끔하게 코드를 작성할 수 있어요! 

```suggestion
export const badge = recipe({
  base: {
    // 버튼 공통 스타일 
  },
  variants: {
    type: {
      Hot: {
        // hot 에 맞는 배경색, 컬러 
      },
      단독판매: {
       // 단독판매 케이스에 맞는 배경색, 컬러  
      },
    },
  },
  defaultVariants: {
    type: 'Hot',
  },
});
```

Reply:
적용 완료!! (❁´◡`❁)